param/_utils.py

```diff
@@ -552,3 +588,22 @@ def _in_ipython():
         return True
     except NameError:
         return False
+
+_RUNNING_TASKS = []
+
+def _cleanup_task(task):
+    if task in _RUNNING_TASKS:
+        _RUNNING_TASKS.remove(task)
+
+def async_executor(func):
+    try:
+        event_loop = asyncio.get_running_loop()
+    except RuntimeError:
+        event_loop = asyncio.new_event_loop()
+    if event_loop.is_running():
+        task = asyncio.ensure_future(func())
+        _RUNNING_TASKS.append(task)
+        task.add_done_callback(_cleanup_task)
```

Comment:
```suggestion
        task.add_done_callback(_RUNNING_TASKS.discard)
```

And remove `_cleanup_task`

Reply:
Confused by this, `_RUNNING_TASKS` is a list. Lists don't have discard.